src/providers/krb5/krb5_common.c

```diff
@@ -390,6 +390,30 @@ errno_t sss_krb5_get_options(TALLOC_CTX *memctx, struct confdb_ctx *cdb,
     return ret;
 }
 
+errno_t sss_krb5_parse_lookahead(const char *param, int *primary, int *backup)
+{
+    int ret;
+    *primary = -1;
+    *backup = -1;
+
+    if (param == NULL)
```

Comment:
please use `{` and `}` even with one-line conditions

Reply:
it might also be a good idea to return an error (or just return in the void version) if primary and backup are NULL